Objective : 
It tests whether the password_valid.py contains checkOther() function or not.

Failed Test :
We check the objective by making an object of Password class in the test_password.py and trying to call the function to check whether the password has no special characters other than !,@,#,$,%,&,*,_.
On doing so we get a failed test message which shows 10 test cases ran and 1 failed with error stating that the class "Password()" does not contain any function called checkOther().
This is due to the fact that our password_valid.py does not contain any such function.

Passed Test :
Code is added in password_valid.py. This code defines a class names "Password()" with one method checkOther(). In the test_password.py file, we import this module and create an object of it and called the function checkOther()
On running the test, the message states that 10 test cases ran and 10 test cases passed with zero errors.



